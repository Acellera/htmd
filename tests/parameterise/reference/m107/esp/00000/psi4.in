set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.655533	 0.008542	 0.000000
C	 -1.668725	 -0.439603	 -0.000000
C	 -0.603259	 0.661624	 -0.000000
H	 -1.578019	 -1.067231	 -0.878235
H	 -1.578019	 -1.067231	 0.878235
C	 0.763820	 0.112474	 -0.000000
H	 -0.707006	 1.295858	 -0.872670
H	 -0.707006	 1.295858	 0.872670
N	 1.813233	 -0.320350	 0.000000

	symmetry c1
}
ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
oeprop('GRID_ESP')
f=open( 'psi4out.xyz', 'w' )
f.write( "9  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
