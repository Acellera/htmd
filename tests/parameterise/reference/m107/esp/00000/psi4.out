    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 16:39:05 2016

    Process ID:  22399
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

H	 -2.655533	 0.008542	 0.000000
C	 -1.668725	 -0.439603	 -0.000000
C	 -0.603259	 0.661624	 -0.000000
H	 -1.578019	 -1.067231	 -0.878235
H	 -1.578019	 -1.067231	 0.878235
C	 0.763820	 0.112474	 -0.000000
H	 -0.707006	 1.295858	 -0.872670
H	 -0.707006	 1.295858	 0.872670
N	 1.813233	 -0.320350	 0.000000

	symmetry c1
}
ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
oeprop('GRID_ESP')
f=open( 'psi4out.xyz', 'w' )
f.write( "9  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-3-16-4.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 16:39:05 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: Cs

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           H         -2.655728204196     0.008587490348     0.000000000000     1.007825032070
           C         -1.668920204196    -0.439557509652     0.000000000000    12.000000000000
           C         -0.603454204196     0.661669490348     0.000000000000    12.000000000000
           H         -1.578214204196    -1.067185509652    -0.878235000000     1.007825032070
           H         -1.578214204196    -1.067185509652     0.878235000000     1.007825032070
           C          0.763624795804     0.112519490348     0.000000000000    12.000000000000
           H         -0.707201204196     1.295903490348    -0.872670000000     1.007825032070
           H         -0.707201204196     1.295903490348     0.872670000000     1.007825032070
           N          1.813037795804    -0.320304509652     0.000000000000    14.003074004780

  Running in c1 symmetry.

  Rotational constants: A =      0.94157  B =      0.15744  C =      0.14190 [cm^-1]
  Rotational constants: A =  28227.64927  B =   4719.89033  C =   4254.08140 [MHz]
  Nuclear repulsion =  103.405162137726776

  Charge       = 0
  Multiplicity = 1
  Electrons    = 30
  Nalpha       = 15
  Nbeta        = 15

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 34
    Number of basis function: 70
    Number of Cartesian functions: 70
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         70      70       0       0       0       0
   -------------------------------------------------------
    Total      70      70      15      15      15       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 141
    Number of basis function: 449
    Number of Cartesian functions: 449
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.3865817439E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -133.96632783569049   -1.33966e+02   8.77678e-02 
   @DF-RHF iter   2:  -110.01668486393807    2.39496e+01   8.02031e-02 DIIS
   @DF-RHF iter   3:  -154.25463676544183   -4.42380e+01   4.91976e-02 DIIS
   @DF-RHF iter   4:  -159.46772128550208   -5.21308e+00   4.62894e-02 DIIS
   @DF-RHF iter   5:  -170.63845460672763   -1.11707e+01   1.15799e-02 DIIS
   @DF-RHF iter   6:  -170.89838225009515   -2.59928e-01   5.66773e-03 DIIS
   @DF-RHF iter   7:  -170.96013091607603   -6.17487e-02   7.29863e-04 DIIS
   @DF-RHF iter   8:  -170.96208227856624   -1.95136e-03   3.25783e-04 DIIS
   @DF-RHF iter   9:  -170.96238718729973   -3.04909e-04   6.86599e-05 DIIS
   @DF-RHF iter  10:  -170.96241938975567   -3.22025e-05   2.85543e-05 DIIS
   @DF-RHF iter  11:  -170.96242914099668   -9.75124e-06   1.36578e-05 DIIS
   @DF-RHF iter  12:  -170.96243211472637   -2.97373e-06   4.13509e-06 DIIS
   @DF-RHF iter  13:  -170.96243235048087   -2.35754e-07   1.18001e-06 DIIS
   @DF-RHF iter  14:  -170.96243236385345   -1.33726e-08   3.48429e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -15.567050     2A    -11.283666     3A    -11.269388  
       4A    -11.245863     5A     -1.224548     6A     -1.082256  
       7A     -0.911762     8A     -0.699353     9A     -0.659434  
      10A     -0.587724    11A     -0.560806    12A     -0.543901  
      13A     -0.536575    14A     -0.458571    15A     -0.458482  

    Virtual:                                                              

      16A      0.195425    17A      0.215157    18A      0.243182  
      19A      0.279581    20A      0.286959    21A      0.308392  
      22A      0.350140    23A      0.362924    24A      0.412366  
      25A      0.462026    26A      0.716576    27A      0.717672  
      28A      0.741109    29A      0.788410    30A      0.805110  
      31A      0.864270    32A      0.892713    33A      0.947506  
      34A      0.978529    35A      1.064269    36A      1.100451  
      37A      1.118116    38A      1.123856    39A      1.148231  
      40A      1.176581    41A      1.177574    42A      1.225111  
      43A      1.345624    44A      1.516865    45A      1.657301  
      46A      1.681496    47A      1.707717    48A      1.789519  
      49A      1.810746    50A      1.919959    51A      1.991380  
      52A      2.132567    53A      2.204211    54A      2.274502  
      55A      2.285826    56A      2.313972    57A      2.339516  
      58A      2.543183    59A      2.580215    60A      2.643719  
      61A      2.663413    62A      2.850104    63A      3.128460  
      64A      3.140344    65A      3.205726    66A      3.643862  
      67A      4.466647    68A      4.528478    69A      4.703973  
      70A      5.070520  

    Final Occupation by Irrep:
              A 
    DOCC [    15 ]

  Energy converged.

  @DF-RHF Final Energy:  -170.96243236385345

   => Energetics <=

    Nuclear Repulsion Energy =            103.4051621377267765
    One-Electron Energy =                -434.5564265445390788
    Two-Electron Energy =                 160.1888320429586088
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -170.9624323638536794

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -6.7799      Y:     0.4378      Z:     0.0000

  Electronic Dipole Moment: (a.u.)
     X:     5.2367      Y:     0.0991      Z:    -0.0000

  Dipole Moment: (a.u.)
     X:    -1.5432      Y:     0.5369      Z:    -0.0000     Total:     1.6340

  Dipole Moment: (Debye)
     X:    -3.9225      Y:     1.3648      Z:    -0.0000     Total:     4.1531


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-3-16-4.cx1.hpc.ic.ac.uk at Tue Jul  5 16:39:09 2016
Module time:
	user time   =       2.87 seconds =       0.05 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          4 seconds =       0.07 minutes
Total time:
	user time   =       2.87 seconds =       0.05 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          4 seconds =       0.07 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -6.7799      Y:     0.4378      Z:     0.0000

  Electronic Dipole Moment: (a.u.)
     X:     5.2367      Y:     0.0991      Z:    -0.0000

  Dipole Moment: (a.u.)
     X:    -1.5432      Y:     0.5369      Z:    -0.0000     Total:     1.6340

  Dipole Moment: (Debye)
     X:    -3.9225      Y:     1.3648      Z:    -0.0000     Total:     4.1531

  Quadrupole Moment: (Debye Ang)
    XX:   -30.4397     YY:   -23.9328     ZZ:   -23.9150
    XY:     1.1775     XZ:     0.0000     YZ:    -0.0000

  Traceless Quadrupole Moment: (Debye Ang)
    XX:    -4.3438     YY:     2.1630     ZZ:     2.1808
    XY:     1.1775     XZ:     0.0000     YZ:    -0.0000

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     H     0.40907  0.40907  0.00000  0.18187
       2     C     3.24293  3.24293  0.00000 -0.48587
       3     C     3.18197  3.18197  0.00000 -0.36394
       4     H     0.40538  0.40538  0.00000  0.18924
       5     H     0.40538  0.40538  0.00000  0.18924
       6     C     2.84850  2.84850  0.00000  0.30299
       7     H     0.38856  0.38856  0.00000  0.22289
       8     H     0.38856  0.38856  0.00000  0.22289
       9     N     3.72965  3.72965  0.00000 -0.45930

   Total alpha = 15.00000, Total beta = 15.00000, Total charge = -0.00000



Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix


 Electrostatic potential computed on the grid and written to grid_esp.dat

*** PSI4 exiting successfully. Buy a developer a beer!
